frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/EnvironmentVariantsModal.tsx

```diff
@@ -273,7 +273,13 @@ export const EnvironmentVariantsModal = ({
         isChangeRequestConfigured(environment?.name || '') &&
         uiConfig.flags.crOnVariants;
 
-    const stickiness = variants[0]?.stickiness || defaultStickiness;
+    const stickiness = useMemo(() => {
+        if (!loading) {
+            return variants[0]?.stickiness || defaultStickiness;
+        }
+        return '';
+    }, [loading, defaultStickiness]);
```

Comment:
shouldn't variants[0] be on the list of memo deps? We're breaking exhaustive deps check rule here.

Reply:
yes - added it in, thanks